etc/profile-m-z/npm.profile

```diff
@@ -0,0 +1,59 @@
+# Firejail profile for npm
+# Description: The Node.js Package Manager
+# This file is overwritten after every install/update
+
+# Persistent local customizations
+include npm.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}/wayland-*
```

Comment:
```suggestion
```

This is redundant due to the `blacklist ${RUNUSER}` below. Removing it makes the profile simpler.